source/recreation.rst

```diff
@@ -70,43 +70,32 @@ If a regression is computed, the model can optionally estimate visitation rates
 Limitations and Simplifications
 -------------------------------
 
-The model does not presuppose that any predictor variable has an effect on visitation.  Instead, the tool estimates the magnitude of each predictor's effect based on its spatial correspondence with current visitation in the area of interest.  The values of photo-person-days per cells are taken as a measure of visitation and are regressed against the values of the predictor variables across all cells.  When a Scenario Predictor Table is provided, the tool uses the :math:`\beta_{p}` values computed for the Predictor Table, and applies them to the future scenario predictors.  This step requires the assumption that people's responses to attributes that serve as predictors in the model will not change over time.  In other words, in the future, people will continue to be attracted to, or repelled by, the predictors in the same way they are currently.
+The model does not presuppose that any predictor variable has an effect on visitation. Instead, the tool estimates the magnitude of each predictor's effect based on its spatial correspondence with current visitation in the area of interest. The values of photo-person-days per cells are taken as a measure of visitation and are regressed against the values of the predictor variables across all cells. When a Scenario Predictor Table is provided, the tool uses the :math:`\beta_{p}` values computed for the Predictor Table, and applies them to the future scenario predictors. This step requires the assumption that people's responses to attributes that serve as predictors in the model will not change over time. In other words, in the future, people will continue to be attracted to, or repelled by, the predictors in the same way they are currently.
 
 
 .. _rec-data-needs: 
 
 Data Needs
 ==========
 
-1. **Workspace (required).** Users must specify a path to the workspace folder where the tool will save its results::
+- :investspec:`recreation.recmodel_client workspace_dir`
 
-     Name: Path to a workspace folder.  Avoid spaces.
-     Sample path: \InVEST\Recreation\
+- :investspec:`recreation.recmodel_client results_suffix`
 
-2. **Results Suffix (optional).** Optionally enter a text string that will be used as a suffix on all output filenames.
+- :investspec:`recreation.recmodel_client aoi_path`
```

Comment:
It's a bit subjective but I wonder if some of that helpful text should end up in the "about" section of the spec or just added in on over here. Maybe:

>It is recommended that this shapefile be projected into linear units, especially if it is used to calculate a regression and scenario. Results are aggregated to these polygons.

Reply:
Added this back in. Should it be marked as `"projected": True` in the args spec? I think I took this out originally because it wasn't. But we don't have to decide now

Reply:
Yea, I'm not certain. But like you say, can handle that in the other related PR. I'm sure Dave will have some insight there.